packages/web-app-files/src/components/SideBar/Links/PublicLinks/LinkInfo.vue

```diff
@@ -117,16 +119,15 @@ export default {
 
     viaRouterParams() {
       const viaPath = this.link.path
-
-      return {
-        name: 'files-personal',
+      // toDo
```

Comment:
Do...what?:)

Reply:
thanks, i already did and forgot to remove the comment :)